pkg/server/backend_manager_test.go

```diff
@@ -29,17 +29,17 @@ type fakeAgentServiceConnectServer struct {
 }
 
 func TestAddRemoveBackends(t *testing.T) {
-	conn1 := new(fakeAgentServiceConnectServer)
-	conn12 := new(fakeAgentServiceConnectServer)
-	conn2 := new(fakeAgentServiceConnectServer)
-	conn22 := new(fakeAgentServiceConnectServer)
-	conn3 := new(fakeAgentServiceConnectServer)
+	conn1 := NewBackend(new(fakeAgentServiceConnectServer))
```

Comment:
Would be better to rename these to backend1 etc given they are no longer conn objects.

Reply:
Done.